src/cobra/summary/model_summary.py

```diff
@@ -396,7 +419,11 @@ def to_string(
         )
 
     def to_html(
-        self, names: bool = False, threshold: float = 1e-6, float_format: str = ".4G"
+        self,
+        names: bool = False,
+        element: str = "C",
+        threshold: float = 1e-6,
```

Comment:
I wonder if we should accept a `None` argument for threshold that initializes it to the model tolerance. This threshold would make little sense if the model tolerance is 1e-3 for instance. Can be left for a future PR though.

Reply:
I would have loved to do that but I wanted to structure the summary in a way that at the time of display, it does not access the model any longer. However, I could copy the model tolerance on initialization and store that value for later.

Reply:
Okay, I have implemented something like what you asked for.